applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -24,1468 +24,1261 @@
-                row( GetGeometry().ShapeFunctionsValues(  ), 
-                point_number ) 
-                );
-            }
-        }
-        else
-        {
-            KRATOS_ERROR << "A constitutive law needs to be specified for the element with ID " << this->Id() << std::endl;
-        }
+    Values.SetStrainVector(this_constitutive_variables.StrainVector);
+    
+    // Reading integration points
+    const GeometryType::IntegrationPointsArrayType& integration_points = GetGeometry().IntegrationPoints(  );
+    
+    // Displacements vector
+    Vector displacements;
+    GetValuesVector(displacements);
+        
+    // Reading integration points
+    for ( unsigned int point_number = 0; point_number < mConstitutiveLawVector.size(); ++point_number ) {
+        // Compute element kinematics B, F, DN_DX ...
+        CalculateKinematicVariables(this_kinematic_variables, point_number, integration_points);
+        
+        // Call the constitutive law to update material variables
+        mConstitutiveLawVector[point_number]->FinalizeMaterialResponse(Values, GetStressMeasure());
         
-        KRATOS_CATCH( "" );
+        mConstitutiveLawVector[point_number]->FinalizeSolutionStep( GetProperties(),
+        GetGeometry(),
+        row( GetGeometry().ShapeFunctionsValues(  ), point_number ),
+        rCurrentProcessInfo 
+        );
     }
+}
 
-    //************************************************************************************
-    //************************************************************************************
+/***********************************************************************************/
+/***********************************************************************************/
 
-    ConstitutiveLaw::StressMeasure BaseSolidElement::GetStressMeasure() const
-    {
-        return ConstitutiveLaw::StressMeasure_PK2;
-    }
-    
-    //************************************************************************************
-    //************************************************************************************
-    
-    void BaseSolidElement::ResetConstitutiveLaw()
-    {
-        KRATOS_TRY
+void BaseSolidElement::InitializeMaterial()
+{
+    KRATOS_TRY
 
-        if ( GetProperties()[CONSTITUTIVE_LAW] != NULL )
-        {
-            for ( unsigned int point_number = 0; point_number < mConstitutiveLawVector.size(); point_number++ )
-                mConstitutiveLawVector[point_number]->ResetMaterial( GetProperties(), 
-                GetGeometry(),                                                                                         row( GetGeometry().ShapeFunctionsValues(  ),                                                                                                                             point_number )
-                );
+    if ( GetProperties()[CONSTITUTIVE_LAW] != nullptr ) {
+        for ( unsigned int point_number = 0; point_number < mConstitutiveLawVector.size(); ++point_number ) {
```

Comment:
Why do you now put all `{` on the same line?
I thought the guideline was to put them on a new line (so the way it was was correct), in fact, you told me at some point to do this

Reply:
In the last meeting we change the criteria (I voted against 👎 )

Reply:
🙄 
Is this documented somewhere?
This can easily be done with clang-format I guess?

Reply:
It was decided in one meeting, not documented, just one issue created on that moment Barcelona Kratos Meeting...